### DISCUSSION TITLE
HTML and Style rendering

### DISCUSSION THREAD

**Discussion by kylelee1027** [reflex-dev/reflex#3161] [Q&A]
When you make an HTML component via rx.html(""), what path does it take to convert the Python code to JavaScript (or something that the browser can process)? Or like what files does it interact with?

**Comment by masenf** [comment]
The Html component is defined here https://github.com/reflex-dev/reflex/blob/main/reflex/components/core/html.py

But i think the issue with the missing styles is due to the tailwind and/or radix css reset being applied. I think in one of the tickets, i suggested the use of the tailwind typography plugin to get around this; maybe that should be the default for the html component?
